caravel/viz.py

```diff
@@ -172,12 +172,9 @@ def get_df(self, query_obj=None):
             raise Exception("No data, review your incantations!")
         else:
             if 'timestamp' in df.columns:
-                if timestamp_format == "epoch_s":
+                if timestamp_format in ("epoch_s", "epoch_ms"):
```

Comment:
The old way is more explicit, but won't work. That's because unix timestamp has been converted to a datetime type, so either `unit='s'` or `unit='m'` will throw error.